libshaderc_spvc/include/spvc/spvc.h

```diff
@@ -335,12 +341,13 @@ SHADERC_EXPORT void shaderc_spvc_set_name(const shaderc_spvc_context_t context,
 SHADERC_EXPORT void shaderc_spvc_build_combined_image_samplers(
     const shaderc_spvc_context_t context);
 
-// For each combined_image_sampler, calls the provided callback function |f|,
-// passing in three arguments read from the combined_image_sampler.
-// (added for GLSL API support in Dawn)
-SHADERC_EXPORT void shaderc_spvc_for_each_combined_image_sampler(
+// Returns the combined image samplers.
+// If |samples| is NULL, then num_samplers is set, and no data is copied.
+// The caller responsible for |samplers| being larger enough to contain all of
```

Comment:
s/The caller responsible/The caller is responsible/

Reply:
Done